### DISCUSSION TITLE
SoupPvP - Suggestions, Bug Reports and Improvement Tips

### DISCUSSION THREAD

**Discussion by Elb1to** [Elb1to/SoupPvP#1] [General]
## 👋 Sup Dawg 😎
This is the place to write about:
 * Bug Reports
 * Suggestions
 * Improvement Tips

Stuff you may want to keep in mind:
 * Don't post anything inappropriate
 * Please be respectful with everyone on here
 * If you report a bug, please make a detailed post about it

**Comment by ohemilyy** [comment]
Suggestion:

Add cock kit

**Reply by Elb1to** [reply]
Maybe B)

**Comment by fabiodimauro-zz** [comment]
Suggestion:

add the spwn and refill commands

spawn : come back to the spawn in X time

refill: refill the player execute the command of soup

**Comment by ohemilyy** [comment]
asdas dasfd asd

**Comment by ohemilyy** [comment]
aaa

**Comment by ohemilyy** [comment]
aaaa

**Comment by ohemilyy** [comment]
aaaa
